__tests__/util/testvariables.js

```diff
@@ -0,0 +1,38 @@
+const login = {
+  fullLogin: 'ruhatest',
```

Comment:
if the object called "login" no need to add *Login to every property. Same for others